tests/testthat/test-timeseries_plot.R

```diff
@@ -0,0 +1,43 @@
+context("timeseries output")
+
+library(RInstatClimatic)
+
+# Get the data --------------------------------------------------------------------------------------------------------
+niger_data <- readRDS("C:/Users/lzc1n17/OneDrive - University of Southampton/PhD/IDEMS/RInstatClimatic/Niger4_day_month_year.RDS")
```

Comment:
Unfortunately, the tests won't pass with files only on your machine.

Is there a dataset from another package we could use instead? Or otherwise just comment this file out for now.

Reply:
Got it. I got this file from an R Instat issue (https://github.com/africanmathsinitiative/R-Instat/issues/6200)

We would want to create examples on these functions. Do we have a data set we can use to put into this R package? Otherwise I can comment it out as you suggest.

Reply:
- `openair` has hourly data
- `tseries` has iceriver daily dataset
- tsdl package has 21 daily sets, most climatic

We can use packages in tests and then include them in `suggests`

Reply:
`tsdl` is not on CRAN (I think). But I've added the others in and changed the test file - thanks!

Reply:
Great thanks!